cmd/pj-rehearse/main.go

```diff
@@ -239,7 +248,7 @@ func rehearseMain() error {
 		prConfig.Prow.ProwJobNamespace = config.StagingNamespace
 	}
 
-	cmClient, err := rehearse.NewCMClient(clusterConfig, prConfig.Prow.ProwJobNamespace, o.dryRun)
+	cmClient, err := rehearse.NewCMClient(clusterConfig, prConfig.Prow.PodNamespace, o.dryRun)
```

Comment:
Is this really that simple? If our pods run at one cluster and prowjobs are in another then pj-rehearse need to work with both, no?

How did this even work with jobs running on build01 before? :/

Reply:
Yeah thats what it does, you can now specify a dedicated kubeconfig for prowjobs.

Are you sure it ever worked with build01?

Reply:
Yeah, sorry, I misread. We always use the cluster config here, and the optionally passed kubeconfig for the pj client. 

From what I'm reading, I would expect this to fail when pj-rehearse runs on a different cluster than the rehearsal (because pj-rehearse always creates this temproary CM on the cluster where it runs itself), but maybe I'm missing something.

But for now this is correct, thanks!